### ISSUE TITLE
How to install tea on nixos

### ISSUE THREAD

**Issue by mbledkowski** [teaxyz/setup#166]
Hi,
How can I install tea cli on NixOS?
Best regards,
Maciej Błędkowski

**Comment by mxcl** [comment]
I don’t understand nixos, and looking into it is clear that if you choose nix you must want to help people who don't to fix things with it. So—over to you.

**Comment by mbledkowski** [comment]
> # sh <(curl tea.xyz)
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100 13952  100 13952    0     0  20221      0 --:--:-- --:--:-- --:--:-- 20220
>                                       /dev/fd/63: line 231: /root/.tea/charm.sh/gum/v0.9.0/bin/gum: No such file or directory
> 
> I checked and this file exists. Does nixos do things differently at a filesystem level even?

Yep, they do a lot of things differently. Basically NixOS is configured through nix config files, and from what I know, you cannot just install a package like that, you have to make nix file for that package.

**Comment by mxcl** [comment]
If we pkg’d pkgx for nixos would any of our packages work though?
